dev/bots/test.sh

```diff
@@ -6,14 +6,28 @@ trap detect_error_on_exit EXIT HUP INT QUIT TERM
 
 detect_error_on_exit() {
     exit_code=$?
-    set +x
+    { set +x; } 2>/dev/null
```

Comment:
Previously, because of the `set -ex` call, "set +x" would be echoed to the console when we ran this line to turn off echoing.  With this change, it won't :-)